main.py

```diff
@@ -7,48 +7,38 @@
 
 import dash
 
-from domain.cached_repo import CachedRepo
+from domain.cached_repo_request_driven import CachedRepoRequestDriven
 from infrastructure.config import ConfigYAML
 from infrastructure.data_access.http.synthetics_repo import SyntheticsRepo
 from presentation.main_view import make_page_layout
 from presentation.matrix_view import make_mesh_test_matrix_layout
 
+FORMAT = "%(asctime)-15s  %(message)s"
+logging.basicConfig(level=logging.DEBUG, format=FORMAT)
```

Comment:
Default log level should not be DEBUG. Make debugging (or log level in general) controllable via configuration.